docs/usage/config-overview.md

```diff
@@ -208,6 +208,43 @@ By default, it is essentially an empty config with only the Renovate JSON schema
 
 If you configure `onboardingConfig` in either Global config or Inherited config then Renovate will use that config directly instead of the default.
 
+If you self-host Renovate in GitLab using [`renovate-runner`](https://gitlab.com/gitlab-com/gl-infra/renovate/renovate-runner), the CI will contain a default [RENOVATE_ONBOARDING_CONFIG](https://gitlab.com/gitlab-com/gl-infra/renovate/renovate-runner/-/blob/d33f9812a326f5496695249f11600591c4d2418b/templates/renovate.gitlab-ci.yml#L5) that will merge with your own configuration settings. For example, the CI by default contains:
```

Comment:
I'm wondering if this documentation should be wholly contained within the runner's own documentation

Reply:
No, I think it's more closely related to the onboarding config. I don't know if this is an issue for other platforms as well. My natural inclination is to search for this info in this location, instead of in the runner's.

Reply:
It is gitlab-runner specific, so I think it should also be present in the runner's own doc.

Reply:
I can open a separate PR there

Reply:
opened https://gitlab.com/renovate-bot/renovate-runner/-/merge_requests/3326

Reply:
agreed.

if we add this info here, then we should link to the official repo, not a mirror

```suggestion
If you self-host Renovate in GitLab using [`renovate-runner`](https://gitlab.com/renovate-bot/renovate-runner), the CI will contain a default [RENOVATE_ONBOARDING_CONFIG](https://gitlab.com/renovate-bot/renovate-runner/-/blob/main#L5) that will merge with your own configuration settings. For example, the CI by default contains:
```

Reply:
fixed